uglify-mangle-excludes.ts

```diff
@@ -10,5 +10,9 @@ export default [
     // Android    
     "DropDownAdapter",
     "DropDownItemSelectedListener",
-    "DropDownTouchListener"
+    "DropDownTouchListener",
+
+    // Angular
```

Comment:
Very possibly not, I actually don't know much about uglify, but saw that you had added all the other named classes in the repo, so assumed it was the same case as Android and Proguard exclusions.